lib/gitter-adapter.js

```diff
@@ -306,6 +307,19 @@ Adapter.prototype.listUsers = function(channel, key) {
   });
 };
 
+Adapter.prototype.autoJoin = function() {
+  this.gitterClient.currentUser()
+  .then(function(user) {
+    return user.rooms();
+  })
+  .then((function(rooms) {
+    rooms.forEach((function(room) {
+      if (room.oneToOne) return;
```

Comment:
They are private messages between users and I was assuming you wouldn't see any new messages if you weren't in the IRC room but it seems to work just fine so your decision seems good to me now.